src/components/loader-spinner/loader-spinner.style.ts

```diff
@@ -0,0 +1,176 @@
+          ${calculateColors("Wrapper", color)};
+        border-radius: 50%;
+      `}
+    `}
+`;
+
+export const StyledLabel = styled(Typography)`
+  display: flex;
+  justify-content: center;
+`;
+
+export const StyledSpinnerCircleSvg = styled.svg<
+  Omit<LoaderSpinnerProps, "showSpinnerLabel" | "labelColor">
+>`
+  ${({ size, tracked, motion, animationTime, color }) =>
+    size &&
+    css`
+      x: 0px;
+      y: 0px;
+
+      circle {
+        fill: transparent;
+        stroke-width: ${LOADER_SPINNER_SIZE_PARAMS[size].strokeWidth}px;
+        stroke: ${calculateColors("Wheel", color)}
+        stroke-linecap: round;
+
+        stroke-dasharray: 100px;
+        stroke-dashoffset: 80px;
+
+        transform-origin: 12px 12px 0px;
+
+        cx: 12px;
+        cy: 12px;
+        r: 10px;
+
+        transform: rotate(270deg);
+
+        @keyframes gradientAnimation {
+          0% {
+            transform: rotate(0deg);
+            stroke-dasharray: 100;
+            stroke: #00D639;
+          }
+
+          33% {
+            stroke: #11AFFF;
+            stroke-dasharray: 90;
+          }
+
+          50% {
+            stroke-dasharray: 80;
+          }
+
+          66% {
+            stroke: #8F49FE;
+            stroke-dasharray: 90;
+          }
+
+          100% {
+            transform: rotate(340deg);
+            stroke-dasharray: 100;
+            stroke: #00D639;
+          }
+        }
+
+        @keyframes trackedAnimation {
+          0% {
+            stroke-dasharray: 100;
+            stroke-dashoffset: 100;
+          }
+
+          100% {
+            stroke-dasharray: 100;
+            stroke-dashoffset: 20;
+          }
+        }
+
+        @keyframes untrackedAnimation {
+          0% {
+            transform: rotate(0deg);
+            stroke-dasharray: 100;
+          }
+
+          20% {
+            transform: rotate(70deg);
+            stroke-dasharray: 90;
+          }
+
+          40% {
+            transform: rotate(140deg);
+            stroke-dasharray: 80;
+          }
+
+          60% {
+            transform: rotate(210deg);
+            stroke-dasharray: 90;
+          }
+
+          80% {
+            transform: rotate(280deg);
+            stroke-dasharray: 100;
+          }
+
+          100% {
+            transform: rotate(360deg);
+            stroke-dasharray: 100;
+          }
+        }
+
+        ${
+          motion &&
+          css`
+            ${color === "gradient-grey" || color === "gradient-white"
+              ? css`
+                  animation: gradientAnimation ${animationTime}s
+                    cubic-bezier(0.2, 0.1, 0.8, 1) infinite;
+                `
+              : css`
+                  animation: ${tracked
+                      ? "trackedAnimation"
+                      : "untrackedAnimation"}
+                    ${animationTime}s cubic-bezier(0.2, 0.1, 0.8, 1) infinite;
+                `}
+          `
+        }
+      }
+    `}
```

Comment:
**suggestion**: separating the arc colour and movement animations as separate sequences should resolve the "jolt" issue when the animation is slowed down:

https://github.com/Sage/carbon/assets/18368713/033ca92c-a5b5-4147-97e2-85f082d30b7e

```suggestion
        ${
          motion &&
          css`
            animation-name: ${tracked
                ? "trackedAnimation"
                : "untrackedAnimation"},
              ${color === "gradient-grey" ||
              (color === "gradient-white" && "gradientAnimation")
                ? "gradientAnimation"
                : "none"};
            animation-duration: ${animationTime}s, 3s;
            animation-timing-function: cubic-bezier(0.2, 0.1, 0.8, 1);
            animation-iteration-count: infinite;
          `
        }
```

I'd suggest making the colour animation duration >=3 seconds as well, to reduce the number of flashes per second.

Reply:
Yeah, the jankiness is slightly more noticeable on the larger sizes, i'll use this approach to make it more seamless.

In terms of the default time - i'll see about maybe 1.5-2 seconds as we'd still like the animation to be fairly quick as its doubtful many loading times will reach 3 seconds - and we'd like end users to be able to see the full animation if possible.